docs/references/failures.mdx

```diff
@@ -87,6 +88,12 @@ When an [Activity Execution](/activities#activity-execution) fails, the Applicat
 When an Activity or Workflow throws an Application Failure, the Failure's `type` field is matched against a Retry Policy's list of [non-retryable errors](/encyclopedia/retry-policies#non-retryable-errors) to determine whether to retry the Activity or Workflow.
 Activities and Workflow can also avoid retrying by setting an Application Failure's `non_retryable` flag to `true`.
 
+### Setting the Next Retry Delay {#next-retry-delay}
+
+By setting the "next retry delay" for a given Application Failure, you can tell the server to wait that amount of time before trying the Activity or Workflow again.  This will override whatever the Retry Policy would have computed for your specific exception.
```

Comment:
is it Next Retry Delay. next retry delay, next retry delay (in quotes) or next_retry_delay? It's inconsistent throughout.